File: go-client-sdk/pkg/models/shared/codatpublicapimodelsrulesalertmodel.go
package shared

import (
	"time"
)

type CodatPublicAPIModelsRulesAlertModel struct {
	AlertID          *string      `json:"alertId,omitempty"`
	ClientID         *string      `json:"clientId,omitempty"`
	ClientName       *string      `json:"clientName,omitempty"`
	CompanyID        *string      `json:"companyId,omitempty"`
	CompanyName      *string      `json:"companyName,omitempty"`
	Data             *interface{} `json:"data,omitempty"`
	DataConnectionID *string      `json:"dataConnectionId,omitempty"`
	ID               *string      `json:"id,omitempty"`
	Message          *string      `json:"message,omitempty"`
	RaisedOnUtc      *time.Time   `json:"raisedOnUtc,omitempty"`
	RuleID           *string      `json:"ruleId,omitempty"`
	RuleType         *string      `json:"ruleType,omitempty"`
}

File: python-client-sdk/src/sdk/models/shared/codat_public_api_models_rules_alertmodel.py
import dataclasses
import dateutil.parser
from dataclasses_json import dataclass_json
from datetime import datetime
from marshmallow import fields
from sdk import utils
from typing import Any, Optional


@dataclass_json
@dataclasses.dataclass
class CodatPublicAPIModelsRulesAlertModel:
    alert_id: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('alertId') }})
    client_id: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('clientId') }})
    client_name: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('clientName') }})
    company_id: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('companyId') }})
    company_name: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('companyName') }})
    data: Optional[Any] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('data') }})
    data_connection_id: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('dataConnectionId') }})
    id: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('id') }})
    message: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('message') }})
    raised_on_utc: Optional[datetime] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('raisedOnUtc'), 'encoder': utils.datetimeisoformat(True), 'decoder': dateutil.parser.isoparse, 'mm_field': fields.DateTime(format='iso') }})
    rule_id: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('ruleId') }})
    rule_type: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('ruleType') }})
    